src/server/game/Instances/InstanceScript.cpp

```diff
@@ -306,6 +307,19 @@ bool InstanceScript::SetBossState(uint32 id, EncounterState state)
                         if (minion->isWorldBoss() && minion->IsAlive())
                             return false;
 
+            switch (state)
+            {
+                case IN_PROGRESS:
+                    SendEncounterStart(1, 0, 1080000, 1080000); // Same values in all packets
```

Comment:
Ye, just found the formular. While I am at it I could also add a battleRezz counter to the group class and use it to add the rezz restriction for the spell effects